app/notification.go

```diff
@@ -333,8 +333,8 @@ func sendNotificationEmail(post *model.Post, user *model.User, channel *model.Ch
 	if *utils.Cfg.EmailSettings.EnableEmailBatching {
 		var sendBatched bool
 		if result := <-Srv.Store.Preference().Get(user.Id, model.PREFERENCE_CATEGORY_NOTIFICATIONS, model.PREFERENCE_NAME_EMAIL_INTERVAL); result.Err != nil {
-			// if the call fails, assume it hasn't been set and don't batch notifications for this user
-			sendBatched = false
+			// if the call fails, assume that the interval has not been explicitly set and batch the notifications
+			sendBatched = true
```

Comment:
The underlying `platform/store/sql_preference_store.go` returns an error if the user hasn't explicitly set this preference, because its `Get(...)` function assumes that there will be one row in the result set, but there are zero. In this case, that isn't an error.